### DISCUSSION TITLE
Mass-assign issues to projects (meaning more than 25 issues...)

### DISCUSSION THREAD

**Discussion by mesqueeb** [community/community#36371] [Projects and Issues]
Even though Github marked **[Mass-assign issues to projects](https://github.com/orgs/community/discussions/6765)** as shipped, it's still not really usable for us...

We need to be able to move all our 1000+ issues to a project: yet there is no way to do this 💥😃🔫

---

The Github Issues UI and the Projects Beta UI to add issues both max out at 25 issues - is there no way around this? Imagine a repo with hundreds of issues.

In the Issues UI, once I select the max amount of issues per page (25), there's no way to select all matching issues (like you can do in, say, Gmail) - you have to paginate page by page to add issues. Furthermore, even if some or all selected issues are already in the project I'm adding to, I still get "N issues have successfully been added to 1 project" even if all N of them are already in the project.

In the Projects Beta UI, there's no way to return more than 25 results. If I do "is:open is:issue", then add those 25 results, pressing Enter again makes the list empty rather than load the next 25. I have to close the Add UI, reopen it, do the search again, then 25 more show up, and rinse and repeat. This is even less user-friendly behavior than the Issue UI one above.

I hope the team can work on improving these aspects of the new Projects Beta.

---

original comment here: https://github.com/orgs/community/discussions/6765#discussioncomment-3141561

**Comment by casella** [comment]
I would also very much appreciate this option. Thanks!
